platform-ui-skin/src/main/webapp/skin/less/core/helpers.less

```diff
@@ -1104,9 +1104,12 @@
     }
   }
   a {
-    color: @blueLinkColor;
+    color: @linkColorUnvisited;
```

Comment:
@akhanfir can we check if blueLinkColor is used only for this context ? can we keep using this same variable blueLinkColor and just modify its color instead of adding a new variable linkColorUnvisited ?

Reply:
if a defined variable is used only in this context, what is the utility of creating it then?
This is one of the reasons why it shouldn't be modified:
http://github.com/Meeds-io/platform-ui/blob/486bcb4490e772614af7ddd1d4e32c946aeeadde/platform-ui-skin/src/main/webapp/skin/less/core/helpers.less#L544-L549